src/toil/job.py

```diff
@@ -1270,6 +1270,8 @@ def _runner(self, jobGraph, jobStore, fileStore):
                filestore
         :return:
         """
+        # store fileStore for use in JobFunctionWrappingJob, ServiceJob, and defer
+        self._fileStore = fileStore
         # Run the job
```

Comment:
Replace with `# ... but also pass it to run() as an argument for backwards compatibility.`